docs/exercises_with_solutions.md

```diff
@@ -2054,13 +2066,36 @@ They were given at week 4 and 7.
     Keywords cannot be used as identifiers. | 
     Each ".cs" file should contain exactly one class. | 
 
+<details><summary>Solution</summary>
+	    Statement | Rule | Convention
+    -----------| ----- | ----- |  
+    Case matters. |			✓			|					
+    Variable names should be descriptive. | 				|			✓		
+    Keywords cannot be used as identifiers. | 			✓			|			
+    Each ".cs" file should contain exactly one class. |					|			✓
+
+</details>
 #. (4 pts) Write a statement that would display, "Hi Mom!" (*with* the quotes) followed by a new line on the screen.
+
+<details><summary>Solution</summary>```Console.WriteLine("\"Hi Mom!\"\n")```</details>
+
 #. (5 pts) Write a series of statements that would 
     #. declare an `int` variable called "myAge",
     #. assign your age to that variable, 
     #. display "My age is ", the value of the "myAge" variable, a period, and finally a new line.
+	
+<details><summary>Solution</summary>
+```
+int myAge;
+myAge = 21;
+Console.WriteLine("My age is " + myAge + ".\n");
+```
+</details>
+	
 #. (Bonus) Give examples of situations where the adage "_Spaces and new lines don't matter in programs_" is actually erroneous.
 
+<details><summary>Solution</summary>Spaces and newlines matter when they are used in string data, as whitespace in strings is formatted exactly how it's typed. Whitespace also matters inbetween words: words in C# must have at least one space between them in order to be compiled correctly (e.g. 'static void Main()' and 'int days = 7'). If there were no spaces in either of the examples, neither of them would compile.</details>
+
```

Comment:
Excellent. Maybe add that they matter for in-line comment?

```
// My comment
int x;
x = 10;
```

If you remove the first newline, the program would not compile.